tap_google_analytics/tap.py

```diff
@@ -85,9 +85,20 @@ class TapGoogleAnalytics(Tap):
         th.Property(
             "reports",
             th.StringType,
-            description="Google Analytics Reports Definition",
+            description="Google Analytics Reports Definition (either a string path or a JSON representation of a list of report objects)",
         ),
         th.Property(
+            "reports_list",
+            th.ArrayType(
+                th.ObjectType(
+                th.Property("name", th.StringType),
+                th.Property("dimensions", th.ArrayType(th.StringType)),
+                th.Property("metrics", th.ArrayType(th.StringType)),
+            )
+        ),
+    description="List of Google Analytics Reports Definitions",
```

Comment:
@nidhi-akkio it looks like the indentation is a little inconsistent here, would you mind reformatting it?